frontend/packages/kubevirt-plugin/console-extensions.json

```diff
@@ -311,7 +317,10 @@
   {
     "type": "console.page/route",
     "properties": {
-      "path": ["/k8s/ns/:ns/virtualmachinesnapshots/:name"],
+      "path": [
+        "/k8s/ns/:ns/virtualmachinesnapshots/:name",
+        "/k8s/ns/:ns/snapshot.kubevirt.io~v1alpha1~VirtualMachineSnapshot/:name"
```

Comment:
Note:
this is only needed when two api versions are installed simultaneously, we will need to remove this ASAP once only one api version is installed